kpatch-build/kpatch-build

```diff
@@ -978,9 +989,10 @@ if [[ -z "$OOT_MODULE" ]]; then
 else
 	KPATCH_BUILD="/lib/modules/$ARCHVERSION/build"
 fi
-KPATCH_BUILD="$KPATCH_BUILD" KPATCH_NAME="$MODNAME" \
+export KPATCH_BUILD="$KPATCH_BUILD" KPATCH_NAME="$MODNAME" \
 KBUILD_EXTRA_SYMBOLS="$KBUILD_EXTRA_SYMBOLS" \
-KPATCH_LDFLAGS="$KPATCH_LDFLAGS" \
+KPATCH_LDFLAGS="$KPATCH_LDFLAGS"
+save_env
```

Comment:
Now the the .env is created in the parent script through the do_export wrapper, it misses this pattern:
`FOO=x my_prog` since `FOO` was not previously exported.  The solution is to export them first.

Reply:
Sure. Those  `save_env` calls may be a moving target if some additional environment is added w/o updating the call, but it's probably easier to just update/move the `save_env` calls if that ever happens.